rotkehlchen/api/v1/schemas.py

```diff
@@ -2867,3 +2868,40 @@ def make_nft_filter_query(
 
 class EventDetailsQuerySchema(Schema):
     identifier = fields.Integer(required=True)
+
+
+class EvmTransactionHashAdditionSchema(AsyncQueryArgumentSchema):
+    evm_chain = EvmChainNameField(required=True, limit_to=list(EVM_CHAIN_IDS_WITH_TRANSACTIONS))
+    tx_hash = EVMTransactionHashField(required=True)
+    associated_address = EthereumAddressField(required=True)
+
+    def __init__(self, db: 'DBHandler') -> None:
+        super().__init__()
+        self.db = db
+
+    @validates_schema
+    def validate_schema(
+            self,
+            data: dict[str, Any],
+            **_kwargs: Any,
+    ) -> None:
+        with self.db.conn.read_ctx() as cursor:
+            tx_count = cursor.execute(
+                'SELECT COUNT(*) FROM evm_transactions WHERE tx_hash=? AND chain_id=?',
+                (data['tx_hash'], data['evm_chain'].value),
+            ).fetchone()
+            if tx_count[0] > 0:
+                raise ValidationError(
+                    message='transaction hash already present in the database',
+                    field_name='tx_hash',
+                )
+
+            is_address_tracked = cursor.execute(
```

Comment:
`is_address_tracked` is a bad name here because it seems that it will be a boolean but is a integer in a tuple, can you rename this variable?